src/Arcus.Messaging.Abstractions/MessageHandling/MessageRouterOptions.cs

```diff
@@ -14,5 +17,10 @@ public class MessageRouterOptions
         /// Gets the options to control the correlation information upon the receiving of messages in the message router.
         /// </summary>
         public MessageCorrelationOptions Correlation { get; } = new MessageCorrelationOptions();
+
+        /// <summary>
+        /// Gets the options to control the Serilog <see cref="MessageCorrelationInfoEnricher"/> when the incoming message is routed via the message router.
+        /// </summary>
+        public MessageCorrelationEnricherOptions Telemetry { get; } = new MessageCorrelationEnricherOptions();
```

Comment:
Not sure about the name though....

Reply:
Telemetry is indeed a weird name, as the property defines some kind of options.
Why not `CorrelationEnricherOptions` instead of `Telemetry`?

Reply:
To be consistent, we would then also need to rename `Correlation` to `CorrelationOptions` ?
Or rename the `Telemetry` property to `CorrelationEnricher` ?

Reply:
It will be good bc it shows the relation with the `Correlation` option. Good suggestion! 👍